source/tutorials/importing.rst

```diff
@@ -123,37 +222,37 @@ The ``fastq.gz`` absolute filepaths may contain environment variables (e.g., ``$
   sample-1,$PWD/some/filepath/sample1_R2.fastq.gz,reverse
   sample-2,$PWD/some/filepath/sample2_R2.fastq.gz,reverse
 
-Just like with ``fastq.gz``, the absolute filepaths in the manifest for any ``fastq`` files must be accurate. The following example illustrates a simple fastq manifest file for ``fastq`` paired-end read data for one sample.
+Just like with ``fastq.gz``, the absolute filepaths in the manifest for any ``fastq`` files must be accurate. The following example illustrates a simple fastq manifest file for ``fastq`` single-end read data for one sample.
```

Comment:
What if we made it two samples? Then there is a bit of symmetry with the psuedo-example above.

Reply:
Yes, I also found this confusing upon first read. That said, I do think it's useful to have an example of single-end reads as well (I can imagine novice users wondering "what do I put for the reverse read if I have single-end reads?)

So I don't feel strongly either way!

Reply:
Oh, good point --- that makes sense to me. How about we leave as-is?

Reply:
Sure!